schema/functions.json

```diff
@@ -90,6 +90,25 @@
         "name",
         "operation"
       ]
+    },
+    "operation": {
+      "oneOf": [
+        {
+          "type": "string",
+          "description": "If type is `openapi`, <path_to_openapi_definition>#<operation_id>. If type is `asyncapi`, <path_to_asyncapi_definition>#<operation_id>. If type is `rpc`, <path_to_grpc_proto_file>#<service_name>#<service_method>. If type is `graphql`, <url_to_graphql_endpoint>#<literal \\\"mutation\\\" or \\\"query\\\">#<query_or_mutation_name>. If type is `odata`, <URI_to_odata_service>#<Entity_Set_Name>. If type is `expression`, defines the workflow expression.",
+          "minLength": 1
+        },
+        {
+          "type": "object",
+          "description": "OpenAPI Path Object definition",
+          "$comment": "https://spec.openapis.org/oas/v3.1.0#paths-object",
```

Comment:
whats the "$" on comment prop. also why hardcode 3.1.0 version here. i think we should find a way to be version independent

Reply:
We have been using schema 7.x since the beginning, and I don't think it will break anything. `$comment` is pretty useful since SDKs/Tooling can use it to parse this data against the OpenAPI spec. So we don't need to replicate their structure here.
 
Additionally, schema 7.x is from [March 2018](https://datatracker.ietf.org/doc/html/draft-handrews-json-schema-01#appendix-B).

Honestly, I don't see why we should not use `$comment` based on this tradeoff.

Reply:
@tsurdilo Any further toughts? If not, please resolve ;)